Update the iPXE instructions.

rcaballeromx opened the pull request:
Signed-off-by: Rodrigo Caballero <rodrigo.caballero.abraham@intel.com>

tpleavitt commented on source/clear-linux/guides/network/ipxe-install.rst:
```diff
@@ -0,0 +1,328 @@
+      pxe_subnet_mask_ip=255.255.255.0
+      pxe_subnet_bitmask=16
+
+#. Add the `pxe-server` bundle to your |CL| system. This bundle has all the
```
Since all commands in this guide are entered as root, I would suggest that you state the the user needs to be logged in as root.

bktan8 replied:
Will do.

bktan8 commented on source/clear-linux/guides/network/ipxe-install.rst:
```diff
@@ -0,0 +1,328 @@
+environment to automatically install |CL|.
+
+The PXE extension known as `iPXE`_\* adds support for additional protocols
+such as HTTP, iSCIS, :abbr:`AoE (ATA over Ethernet)`, and
```
You're right Tom!  It's iSCSI.
will update.

rcaballeromx commented on source/clear-linux/guides/network/ipxe-install.rst:
```diff
@@ -0,0 +1,328 @@
+:abbr:`FCoE (Fiber Channel over Ethernet)`. iPXE can also be used to enable
+network booting on computers which lack built-in PXE support.
+
+Figure 1 depicts the flow of information between a PXE server and a PXE
```
No, "a PXE Client" is correct, since there can be more than one client and the sentence is not referring to a specific client.

rcaballeromx commented:
I kept all of the changes @bktan8 had made, fixed an indentation issue causing the numbering to restart, and removed an unnecessary note directive. Could you take a look @tpleavitt?
@iphutch: Can you merge once Tom approves?
